packaging/systemd/configure-ovs.sh

```diff
@@ -626,6 +626,23 @@ fi
 # print initial state
 print_state
 if [ "$1" == "OVNKubernetes" ]; then
+  # Skip configuring NICs onto OVS bridge "br-ex" when disableOVSInit is true
+  MICROSHIFT_CONFIG_DIRS=(~/.microshift /etc/microshift)
+  MICROSHIFT_CONFIG_FILE_NAME="config.yaml"
+  MICROSHIFT_OVN_CONFIG_FILE_NAME="ovn.yaml"
+  for dir in "${MICROSHIFT_CONFIG_DIRS[@]}"; do
+    if [ -f "$dir/$MICROSHIFT_CONFIG_FILE_NAME" ] && [ -f "$dir/$MICROSHIFT_OVN_CONFIG_FILE_NAME" ]; then
+      disableOVSInit=$(cat "$dir/$MICROSHIFT_OVN_CONFIG_FILE_NAME" | awk "/disableOVSInit:/ && ! /#.*disableOVSInit:/ {print \$2}")
```

Comment:
Can we use yq to parse the YAML? How big is it and is it OK as a dependency?

If not, would it make sense to use a file format that is friendly for parsing with bash?

Reply:
I believe @zshi-redhat solution is OK, we avoid installing yq which based on his comments doesn't seem to be available in RHEL, and we keep yaml which is a more familiar config format in the k8s world, also more human friendly than json.